python/ray/autoscaler/azure/azure_dev_deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
do you really want to push this to ray?

Reply:
it will be lost otherwise... we can strip out the names and subscription info if you want.

Reply:
i think it might be better in a readme describing a workflow for using a jump box and listing the commands needed. I would remove the blob / wheel pieces though, they shouldn't be relevant after this is merged and new wheels are built